mbedtls/src/pk/mod.rs

```diff
@@ -976,7 +870,7 @@ impl Pk {
 
     pub fn write_private_der_vec(&mut self) -> Result<Vec<u8>> {
         crate::private::alloc_vec_repeat(
-            |buf, size| unsafe { pk_write_key_der(&mut self.inner, buf, size) },
+            |buf, size| { let _ = &self; unsafe { pk_write_key_der(&mut self.inner, buf, size) } },
```

Comment:
Why do you need the `let _ = &self` ?

Reply:
This are some dummy code added when I run `cargo fix --edition` to fix some syntax  error.
Will remove them

Reply:
removed